package.json

```diff
@@ -80,6 +79,7 @@
     "copy-to-clipboard": "^3.3.1",
     "dotenv": "^16.0.1",
     "easy-peasy": "^5.0.4",
+		"electron": "^19.0.8",
```

Comment:
ESLint said to list electron as dependency instead of dev-dependency, so I manually changed its position in package.json.